components/automate-chef-io/data/docs/api-static/03-tag-groups.swagger.json

```diff
@@ -9,7 +9,11 @@
         "Compliance Reporting Profiles",
         "Compliance Reporting Report IDs",
         "Compliance Reporting Reports",
-        "Compliance Reporting Suggestions"
+        "Compliance Reporting Suggestions",
+        "Compliance Stats Summary",
+        "Compliance Stats Trend",
+        "Compliance Stats Profiles",
+        "Compliance Stats Failures"
```

Comment:
This makes me realize there is some cruft / complexity around how we're handling tags and groups due to learning how they worked. I'll add it to the TODO list.

Reply:
ya i kinda just added those cause i saw there were some for reporting. not actually sure what they do! 😺

Reply:
I handled it in the 00 file, which looks like it overrides this file.